lib/features/home/view/widgets/property_item_view.dart

```diff
@@ -0,0 +1,376 @@
+                            .map((photo) => ValueListenableBuilder<int>(
+                                valueListenable: _idxNotifier,
+                                builder: (_, selectedIndex, __) {
+                                  return Container(
+                                    margin: EdgeInsets.only(
+                                        right: widget.photos.indexOf(photo) ==
+                                                widget.photos.length - 1
+                                            ? 0
+                                            : HsDimens.spacing4),
+                                    child: _DotIndicator(
+                                      isSelected: selectedIndex ==
+                                          widget.photos.indexOf(photo),
+                                    ),
+                                  );
+                                }))
+                            .toList(),
+                      ),
+                      const SizedBox(
+                        height: HsDimens.spacing8,
+                      ),
+                      Container(
+                        padding: const EdgeInsets.all(HsDimens.spacing4),
+                        decoration: BoxDecoration(
+                            color: HSColor.background.withOpacity(0.9),
+                            borderRadius: BorderRadius.circular(7)),
+                        child: Row(
+                          children: [
+                            CircleAvatar(
+                              radius: HsDimens.spacing20,
+                              backgroundImage: NetworkImage(widget.ownerAvatar),
+                            ),
+                            const SizedBox(
+                              width: HsDimens.spacing4,
+                            ),
+                            Expanded(
+                              child: Text(
+                                widget.ownerName,
+                                maxLines: 2,
+                                overflow: TextOverflow.ellipsis,
+                                style: Theme.of(context)
+                                    .textTheme
+                                    .bodySmall
+                                    ?.copyWith(
+                                        color: HSColor.neutral6,
+                                        fontWeight: FontWeight.w500),
+                              ),
+                            ),
+                            const SizedBox(width: 12),
+                            Expanded(
+                                child: RichText(
+                                    textAlign: TextAlign.end,
+                                    maxLines: 4,
+                                    overflow: TextOverflow.ellipsis,
+                                    text: TextSpan(
+                                        text:
+                                            '${HatSpaceStrings.current.availableDate}: ',
```

Comment:
This is a string, why do u need to convert it to string again?

Please include ':' in the string in arb file